ssr/src/page/post_view/overlay.rs

```diff
@@ -112,7 +112,10 @@ fn LikeAndAuthCanLoader(post: PostDetails) -> impl IntoView {
 }
 
 #[component]
-pub fn VideoDetailsOverlay(post: PostDetails) -> impl IntoView {
+pub fn VideoDetailsOverlay(
+    post: PostDetails,
+    set_eligible_onboarding_post: Option<WriteSignal<bool>>,
```

Comment:
use context instead of prop drilling like this (Hint: PostViewCtx)

Reply:
Done